beetsplug/spotify.py

```diff
@@ -164,6 +164,12 @@ def _handle_response(self, request_type, url, params=None):
                 )
                 self._authenticate()
                 return self._handle_response(request_type, url, params=params)
+            elif response.status_code == 429:
+                seconds = response.headers['Retry-After']
+                time.sleep(int(seconds))
+                self._log.info('Too many API requests. Retrying after {} \
```

Comment:
My thought was to keep the user informed about the delay. It will also be nice to get those reports if users see those messages so that we can adjust the timeout. In any case, I have changed it to `debug`.

Reply:
It looks like this changed back to `info`?